src/lib/expansion/expansion-panel.scss

```diff
@@ -5,13 +5,13 @@
 .mat-expansion-panel {
   $border-radius: 4px;
 
-  @include mat-elevation-transition;
   @include mat-overridable-elevation(2);
   box-sizing: content-box;
   display: block;
   margin: 0;
-  transition: margin 225ms $mat-fast-out-slow-in-timing-function;
   border-radius: $border-radius;
+  transition: margin 225ms $mat-fast-out-slow-in-timing-function,
+      box-shadow $mat-elevation-transition-duration $mat-elevation-transition-timing-function;
```

Comment:
You should consider using the SaSS function: mat-elevation-transition-property-value. It's being used for exact same situations by other components IIRC.

Reply:
Done.